### DISCUSSION TITLE
Publicly available plugins

### DISCUSSION THREAD

**Discussion by bitmap4** [nukeop/nuclear#1086] [Q&A]
Are there any publicly available user plugins that I can use in the user plugins tab?
![Screenshot 2021-11-06 at 12 22 43 PM](https://user-images.githubusercontent.com/64366765/140600879-46eef80d-2c07-4c7d-8b4c-283321ddb852.png)

**Comment by Venryx** [comment]
I created one: https://github.com/Venryx/nuclear-vplugin

Basically, it lets you set custom "weights" for different folders in your library (on the tree-view layout of the library panel), and then have it generate a playlist based on it.

![](https://i.imgur.com/ZINtWkU.png)

However, it's not really intended for other people to use atm as the UI is very rough (though it gets the job done for me). I mention it just as an example of something you can use plugins for.

**Reply by vinnymckay33** [reply]
I second your query @BitMap4
